samples/test/translate_model_management_test.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * Copyright 2019 Google LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const {assert} = require('chai');
+const {AutoMlClient} = require('@google-cloud/automl');
+
+const cp = require('child_process');
+
+const execSync = cmd => cp.execSync(cmd, {encoding: 'utf-8'});
+
+// const DATASET_ID = 'TRL8522556519449886720';
```

Comment:
how hard is it to bootstrap these tests so that they can be run locally, would love to figure out a way to generalize the process of bootstrapping a dataset, so that it's easy for someone to test locally.

Reply:
External or Googler?